src/cli/serve/serve.js

```diff
@@ -237,7 +243,7 @@ export default function (program) {
           silent: !!opts.silent,
           watch: !!opts.watch,
           repl: !!opts.repl,
-          basePath: !!opts.basePath,
+          basePath: opts.runExamples ? false : !!opts.basePath,
```

Comment:
What's the reason for this?

Reply:
Good question. Added an explanation.  Copying here:

```
          // We want to run without base path when the `--run-examples` flag is given so that we can use local
          // links in other documentation sources, like "View this tutorial [here](http://localhost:5601/app/tutorial/xyz)".
          // We can tell users they only have to run with `yarn start --run-examples` to get those
          // local links to work.  Similar to what we do for "View in Console" links in our
          // elastic.co links.
```